src/components/ToggleSwitch.js

```diff
@@ -11,6 +11,7 @@ const StyleUtils = require('../utils/Style');
 
 class ToggleSwitch extends React.Component {
   static propTypes = {
+    ariaLabel: PropTypes.string, 
```

Comment:
Do you think it would be worth the time to add a more generic 'pass through' prop like `buttonProps` that get spread onto the button? That way we don't have to keep adding individual props like this.

Reply:
Yeah I had thought to do that but went this way on the first run. I like the idea of the generic pass through. I will get that implemented.